src/gui/search/searchsortmodel.cpp

```diff
@@ -47,8 +119,47 @@ bool SearchSortModel::lessThan(const QModelIndex &left, const QModelIndex &right
         if (Utils::String::naturalSort(vL.toString(), vR.toString(), res))
             return res;
 
-        return QSortFilterProxyModel::lessThan(left, right);
+        return base::lessThan(left, right);
+    }
```

Comment:
You can move the closing brace one line above and have the same base return outside the brace.